pkg/service/utgen/injector.go

```diff
@@ -155,26 +155,23 @@ func (i *Injector) updateJavaScriptImports(importedContent string, newImports []
 	existingImportsSet := make(map[string]bool)
 
 	existingImports := importRegex.FindAllString(importedContent, -1)
+	allImports := make([]string, 0)
 	for _, imp := range existingImports {
-		if imp != "\"\"" && len(imp) > 0 {
-			existingImportsSet[imp] = true
+		trimmedImp := strings.TrimSpace(imp)
+		if len(trimmedImp) > 0 {
+			existingImportsSet[trimmedImp] = true
 		}
+		allImports = append(allImports, imp)
 	}
-
 	for _, imp := range newImports {
 		imp = strings.TrimSpace(imp)
-		if importRegex.MatchString(imp) {
+		if !existingImportsSet[imp] && importRegex.MatchString(imp) {
 			existingImportsSet[imp] = true
+			allImports = append(allImports, imp)
 		}
 	}
 
-	allImports := make([]string, 0, len(existingImportsSet))
-	for imp := range existingImportsSet {
-		allImports = append(allImports, imp)
-	}
-
 	importSection := strings.Join(allImports, "\n")
-
 	updatedContent := importRegex.ReplaceAllString(importedContent, "")
```

Comment:
hey @sup194 here is an observation this code would remove all the space and hence new tests files are added often in wrong line.
refer <img width="730" alt="image" src="https://github.com/user-attachments/assets/b4e12d24-60f6-4586-bb40-d9cebc77f1cf">


In go update import I've respected the presence of spaces in function extractGoImports, so that when writing imports or requirements the old spaces exist, and since we just have new import lines we ensure that new tests have the correct import length

Reply:
Additionally, while working on modifying the Python import logic, I noticed similar issues. The old method for handling Python imports was unable to pass the new Python import unit tests I added, indicating that there might be some underlying problems with the existing approach too.
Could you clarify whether this is the case, or if there are spectific expectations around how import formatting (e.g., spaces) should be handled?

Reply:
agreed @sup194 its an existing problem which I solved in the goimports ..

this was an existing issue the spaces were always removed which I fixe in Go, basically in import section of code I leave it as it is, and just add new import below the old imports(ensuring if the installs have space we won't alter it)

Reply:
```
const { isEven } = require('../src/controller');
const { square } = require('../src/controller');
const { dovode } = require('../src/controller');
const { max } = require('../src/controller');
const { sum } = require('../src/controller');
const { modulus } = require('../src/controller');
const { multiply } = require('../src/controller');
const { isOdd } = require('../src/controller');
const { factorial } = require('../src/controller');
const { min } = require('../src/controller');
const { cube } = require('../src/controller');
const { subtraction } = require('../src/controller');
const { isPrime } = require('../src/controller');
describe('Dummy test', () => {
    it('dummy test', async () => {
        expect(true);
    });
});
// Test generated by Keploy 🐰
test('sum of 2 and 3 should be 5', () => {
  expect(sum(2, 3)).toBe(5);
});
// Test generated by Keploy 🐰
test('subtraction of -5 and -3 should be -2', () => {
  expect(subtraction(-5, -3)).toBe(-2);
});
// Test generated by Keploy 🐰
test('multiplication of 5 and 0 should be 0', () => {
  expect(multiply(5, 0)).toBe(0);
});
// Test generated by Keploy 🐰
test('7 should be a prime number', () => {
  expect(isPrime(7)).toBe(true);
});
// Test generated by Keploy 🐰
test('4 should be even', () => {
  expect(isEven(4)).toBe(true);
});
// Test generated by Keploy 🐰
test('factorial of 5 should be 120', () => {
  expect(factorial(5)).toBe(120);
});
// Test generated by Keploy 🐰
test('division of 10 by 2 should be 5', () => {
  const { dovode } = require('../src/controller');
  expect(dovode(10, 2)).toBe(5);
});
// Test generated by Keploy 🐰
test('modulus of 10 and 3 should be 1', () => {
  const { modulus } = require('../src/controller');
  expect(modulus(10, 3)).toBe(1);
});
// Test generated by Keploy 🐰
test('maximum of 10 and 3 should be 10', () => {
  const { max } = require('../src/controller');
  expect(max(10, 3)).toBe(10);
});
// Test generated by Keploy 🐰
test('minimum of 10 and 3 should be 3', () => {
  const { min } = require('../src/controller');
});
// Test generated by Keploy 🐰
test('5 should be odd', () => {
  const { isOdd } = require('../src/controller');
  expect(isOdd(5)).toBe(true);
});
// Test generated by Keploy 🐰
test('5 should be odd', () => {
  const { isOdd } = require('../src/controller');
  expect(isOdd(5)).toBe(true);
});
// Test generated by Keploy 🐰
test('minimum of 10 and 3 should be 3', () => {
  const { min } = require('../src/controller');
  expect(min(10, 3)).toBe(3);
});
// Test generated by Keploy 🐰
test('square of 4 should be 16', () => {
  const { square } = require('../src/controller');
  expect(square(4)).toBe(16);
});
// Test generated by Keploy 🐰
test('cube of 3 should be 27', () => {
  const { cube } = require('../src/controller');
  expect(cube(3)).toBe(27);
});
```

currently if the file is the above one

it leads to 
```
const { min } = require('../src/controller');
const { celsiusToFahrenheit } = require('../src/controller');
const { sum } = require('../src/controller');
const { modulus } = require('../src/controller');
const { cube } = require('../src/controller');
const { isEven } = require('../src/controller');
const { dovode } = require('../src/controller');
const { max } = require('../src/controller');
const { multiply } = require('../src/controller');
const { isPrime } = require('../src/controller');
const { square } = require('../src/controller');
const { isOdd } = require('../src/controller');
const { factorial } = require('../src/controller');
const { subtraction } = require('../src/controller');
describe('Dummy test', () => {
    it('dummy test', async () => {
        expect(true);
    });
});
// Test generated by Keploy 🐰
test('sum of 2 and 3 should be 5', () => {
  expect(sum(2, 3)).toBe(5);
});
// Test generated by Keploy 🐰
test('subtraction of -5 and -3 should be -2', () => {
  expect(subtraction(-5, -3)).toBe(-2);
});
// Test generated by Keploy 🐰
test('multiplication of 5 and 0 should be 0', () => {
  expect(multiply(5, 0)).toBe(0);
});
// Test generated by Keploy 🐰
test('7 should be a prime number', () => {
  expect(isPrime(7)).toBe(true);
});
// Test generated by Keploy 🐰
test('4 should be even', () => {
  expect(isEven(4)).toBe(true);
});
// Test generated by Keploy 🐰
test('factorial of 5 should be 120', () => {
  expect(factorial(5)).toBe(120);
});
// Test generated by Keploy 🐰
test('division of 10 by 2 should be 5', () => {
  const { dovode } = require('../src/controller');
  expect(dovode(10, 2)).toBe(5);
});
// Test generated by Keploy 🐰
test('modulus of 10 and 3 should be 1', () => {
  const { modulus } = require('../src/controller');
  expect(modulus(10, 3)).toBe(1);
});
// Test generated by Keploy 🐰
test('maximum of 10 and 3 should be 10', () => {
  const { max } = require('../src/controller');
  expect(max(10, 3)).toBe(10);
});
// Test generated by Keploy 🐰
test('minimum of 10 and 3 should be 3', () => {
  const { min } = require('../src/controller');
});
// Test generated by Keploy 🐰
test('5 should be odd', () => {
  const { isOdd } = require('../src/controller');
  expect(isOdd(5)).toBe(true);
});
// Test generated by Keploy 🐰
test('5 should be odd', () => {
  const { isOdd } = require('../src/controller');
  expect(isOdd(5)).toBe(true);
});
// Test generated by Keploy 🐰
test('minimum of 10 and 3 should be 3', () => {
  const { min } = require('../src/controller');
  expect(min(10, 3)).toBe(3);
});
// Test generated by Keploy 🐰
test('square of 4 should be 16', () => {
  const { square } = require('../src/controller');
  expect(square(4)).toBe(16);
});
// Test generated by Keploy 🐰
test('cube of 3 should be 27', () => {
  const { cube } = require('../src/controller');
  expect(cube(3)).toBe(27);
// Test generated by Keploy 🐰
test('celsiusToFahrenheit of 0 should be 32', () => {
  const { celsiusToFahrenheit } = require('../src/controller');
  expect(celsiusToFahrenheit(0)).toBe(32);
});
});
```


Thus a better optimised approach would fix this kind of issues

Reply:
Got it, thanks for the clarification. I'll fix it

Reply:
@shivamsouravjha Hi, I’ve updated the implementation as discussed. Could you please take a moment to review it when you have time? Thanks!